public/src/components/channelManagement/TestMethodologyEditor.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { Methodology } from './helpers/shared';
+import { makeStyles } from '@mui/styles';
+import {
+  Button,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  TextField,
+  Theme,
+  Tooltip,
+} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import AddIcon from '@mui/icons-material/Add';
+
+const useStyles = makeStyles(({ spacing, palette }: Theme) => ({
+  container: {
+    '& > * + *': {
+      marginTop: spacing(1),
+    },
+  },
+  methodologyContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    border: `1px solid ${palette.grey[800]}`,
+    borderRadius: '4px',
+    padding: spacing(1),
+    '& > * + *': {
+      marginLeft: spacing(1),
+    },
+  },
+  audiencePercentage: {
+    display: 'flex',
+    alignItems: 'center',
+    margin: `0 ${spacing(1)}`,
+    fontWeight: 500,
+  },
+  testNameAndDeleteButton: {
+    marginLeft: 'auto',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  deleteButton: {
+    '& > button': {
+      height: '100%',
+    },
+  },
+  testName: {
+    marginRight: spacing(2),
+    fontWeight: 500,
+  },
+  error: {
+    color: 'red',
+  },
+}));
+
+const defaultEpsilonGreedyBandit: Methodology = {
+  name: 'EpsilonGreedyBandit',
+  epsilon: 0.1,
```

Comment:
Do we need any documentation to help support setting the epsilon value? Or do marketing already have that knowledge?

Reply:
👍 we can document it in the RRCP user guide